src/main/java/org/elasticsearch/action/percolate/PercolateShardResponse.java

```diff
@@ -112,6 +118,10 @@ public InternalAggregations aggregations() {
         return aggregations;
     }
 
+    public List<SiblingReducer> reducers() {
+        return reducers;
+    }
```

Comment:
Added a percolator test and implemented read/write methods correctly to serialise the reducers